app/controllers/valid_attachment_types_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ValidAttachmentTypesController < ApplicationController
```

Comment:
this should probably be renamed now that we're changing the way it's used from the original